systemvm/patches/debian/config/opt/cloud/bin/cs/CsConfig.py

```diff
@@ -82,7 +82,10 @@ def get_dns(self):
         conf = self.cmdline().idata()
         dns = []
         if not self.use_extdns():
-            dns.append(self.address().get_guest_ip())
+            if not self.is_vpc() and self.cl.is_redundant() and self.cl.get_guest_gw():
```

Comment:
so this would treat isolated and shared networks alike. Is that acceptable @ustcweizhou ?